docs/com.redhat.viaq-openshift-project.asciidoc

```diff
@@ -197,6 +203,16 @@ to correlate data with the data stored in Elasticsearch, this field
 will give you the exact document corresponding to the record.
 
 
+==== viaq_index_name
+
+type: keyword
+
+example: project.my-cool-project-in-lab04.748e92c2-70d7-11e9-b387-000d3af2d83b.2019.05.09
```

Comment:
same here.  do we need to consider obsoleting these files in favor of a different set?

Reply:
Once is ViaQ/elasticsearch-templates#118 is merged and a new release is done than all places where this field is documented will be fixed in one shot.